data/mods/Magiclysm/enchanted/enchanted_belts.json

```diff
@@ -0,0 +1,186 @@
+[
+  {
+    "abstract": "mbelt_leather",
+    "type": "TOOL_ARMOR",
+    "name": "magic leather belt",
+    "description": "A leather belt.  Useful for making your pair of pants fit.",
+    "weight": "102 g",
+    "volume": "250 ml",
+    "price": 90000,
+    "bashing": 5,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 5,
+    "material_thickness": 2,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Stick what into your belt",
+      "holster_msg": "You tuck your %s into your %s",
+      "max_volume": "500 ml",
+      "max_weight": 400,
+      "draw_cost": 60,
+      "flags": [ "BELT_CLIP" ]
+    },
+    "flags": [ "WAIST", "WATER_FRIENDLY", "STURDY" ]
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_haste",
+    "name": "Belt of Haste",
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "multiply": 0.15, "add": 5 } ] } ]
+    }
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_thor",
+    "name": "Megingjörð",
+    "description": "The mythical belt of Thor, god of thunder.  Or at least so it appears.  It doubles the wearer's base strength.",
+    "material": [ "superalloy" ],
+    "material_thickness": 10,
+    "environmental": 20,
+    "weight_capacity_bonus": "50 kg",
```

Comment:
doesn't this stack with the strength? that's quite a bit of extra carrying capacity. of course this appears to be a legendary item, so maybe it's warranted if you have to go through a questline or dangerous dungeon to get it.

Reply:
I was iffy about that myself, and only added it on after I saw that there WAS a such thing as weight_capacity_bonus, but it makes sense. I'd rather put say lifting 1, but I haven't tried putting that on an item.